flexible-classroom/overview/_category_.json

```diff
@@ -0,0 +1,6 @@
+{
+  "position": 1,
+  "label": "Overview",
```

Comment:
Suggestion: I noticed that you cut out the current product overview (https://docs.agora.io/en/agora-class/product_agora_class?platform=iOS) and features (https://docs.agora.io/en/agora-class/agora_class_prod_archit?platform=iOS) documentation. This is the part that the US DE and product manager specifically asked to be added to the documentation. If this content is missing, new users will not be able to understand the uniqueness of the product as an aPaaS product and other product details. Can we add this content back in?

Reply:
The simple product overview is standard for all products now. I've added the diagram to the  enable/configure page. We can have a think about this after release.